src/controllers/update/confirm.overseas.entity.details.controller.ts

```diff
@@ -15,12 +16,21 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     const appData: ApplicationData = getApplicationData(req.session as Session);
     const update = appData.update as Update;
 
+    let journeyQuery = "";
+    let journeyType = config.JourneyType.update;
+    if (isRemoveJourney(req)) {
+      journeyType = config.JourneyType.remove;
+      journeyQuery = "?joureny=remove";
+    }
+
     return res.render(config.CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE, {
-      backLinkUrl: config.OVERSEAS_ENTITY_QUERY_URL,
+      backLinkUrl: `${config.OVERSEAS_ENTITY_QUERY_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`,
```

Comment:
i would duplicate the res.render so we leave the original one untouched, but have another res.render if isRemoveJourney(req) is true